### DISCUSSION TITLE
Bad Request 400 when using language:German, model:general and tier:nova

### DISCUSSION THREAD

**Discussion by sceiler** [deepgram/community#135] [General help] [answered]
I am currently trying out Deepgram with the Obsidian plugin (https://github.com/jjmaldonis/obsidian-audio-notes).
My plugin settings are:
Language: German
Tier: Nova

The resulting API request is: `POST /v1/listen?keywords=&language=de&model=general&numbers=true&profanity_filter=true&punctuate=true&tier=nova` however that results in a `Error: "Bad Request: No such model/language found."`

According to this documentation https://developers.deepgram.com/api-reference/transcription/#transcribe-pre-recorded-audio the values in the POST request should be valid.

Does anyone know why this fails?

**Comment by briancbarrow** [comment, marked as answer]
Hi @sceiler,

Nova is not available with German right now. You can reference the [Language documentation](https://developers.deepgram.com/documentation/features/language/) for more information about which languages are available on which tiers/models.
